def test_func():
    raise Exception("hello")


def test_a():
    print('hi')
